package.json

```diff
@@ -57,11 +57,16 @@
     "chai-uuid": "^1.0.6",
     "dotenv": "6.2.0",
     "express": "4.16.4",
+    "express-correlation-id": "^1.3.0",
+    "express-winston": "^3.2.1",
     "joi": "13.7.0",
+    "morgan": "^1.9.1",
```

Comment:
a to nie mówiłeś że jest wybór morgan vs winston? bo teraz obie są w zależnościach?